game/missiongenerator/aircraft/flightgroupconfigurator.py

```diff
@@ -201,7 +201,7 @@ def skill_level_for(self, unit: FlyingUnit, pilot: Optional[Pilot]) -> Skill:
         missions_for_skill_increase = 4
         increase = pilot.record.missions_flown // missions_for_skill_increase
         capped_increase = min(current_level + increase, len(levels) - 1)
-        new_level = (capped_increase, current_level)[
+        new_level = (current_level, capped_increase)[
```

Comment:
We probably wouldn't have made this mistake if this was just a regular conditional. I'll go ahead and merge this since the status quo isn't your fault and this is definitely an improvement, but if you want to send another PR doing that trivial cleanup I'd merge that too :)

Reply:
Sure, follow up PR using regular conditional can be found here: https://github.com/dcs-liberation/dcs_liberation/pull/2350 If the condition can be shortened more elegantly let me know. Typically write in Java and would have used an ? assignment.

Reply:
Thanks, merged that too.

Just so you know if you ever want it, the python equivalent of `foo ? bar : baz` is `bar if foo else baz`. I think the multi-line conditional reads perfectly fine here though.